```{r, echo=FALSE, eval=TRUE, warning=FALSE, message=FALSE}
opts_chunk$set("dev" = c("png", "cairo_pdf"))
opts_chunk$set(results = "hold")
opts_chunk$set(echo = FALSE)
opts_chunk$set(fig.show = "hold")
opts_chunk$set(warning = FALSE)
opts_chunk$set(fig.align = "center")
opts_chunk$set(cache = TRUE)

library("randomForest", quietly=TRUE)
library("png", quietly=TRUE)
```

**Manipulation of the Gut Microbiome Reveals Role for Microbial Community Structure in Colon Tumorigenesis**

Joseph P. Zackular<sup>1</sup>, Nielson T. Baxter<sup>1</sup>, Grace Y.
Chen<sup>2</sup>, and Patrick D. Schloss<sup>1*</sup>

\* To whom correspondence should be addressed.

<pschloss@umich.edu>

1 Department of Microbiology and Immunology, University of Michigan, Ann
Arbor, MI

2 Department of Internal Medicine, Division of Hematology and Oncology,
University of Michigan, Ann Arbor, MI


\newpage

**Abstract**

Abstract will go here

\newpage

**Introduction**

The mammalian gastrointestinal tract is home to a complex and dynamic community
of microorganisms, termed the gut microbiome, which is essential for maintaining
host health [Backhed et al*.,* 2005]. Among the bacterial populations that make
make up the gut microbiome there are complex interactions that have an important
effect on host health [Levy and Bordenstein, 2013, Marino et al., 2013, Lepp et
al., 2004]. The number of diseases that are associated with abnormalities in the
gut microbiome highlights the importance of these ecological interactions
[Turnbaugh et al. 2006; Tamboli et al.; 2004, Saulnier et al., 2011]. Over the
last several years it has been well documented that perturbations to this
community are associated with colorectal cancer (CRC) in humans and mice [Chen
et al., 2013; Chen, et al., 2012; Kostic et al., 2012; Geng et al., 2013; Shen
et al., 2013; Sobhani et al., 2011; Wang et al., 2012; Ahn et al., 2013]. We
have previously shown that CRC-associated changes in the gut microbiome directly
potentiate colon tumorigenesis in a mouse model of CRC [Zackular et al., 2013].
In that study we observed clear shifts in the microbiome that could be
associated with a stepwise progression in the number of tumors that were found
in the colon. In addition, we showed that transfer of the tumor-associated
microbiota to germ-free mice resulted in increased tumor formation relative to
germ-free mice receiving the microbiota of healthy mice. These results were
supported by a subsequent study where we colonized germ-free mice with the
microbiota of human donors and observed significant variation in the number of
tumors that formed [Baxter et al., 2014]. Combined, these results demonstrate
that the microbiota interact with the host to potentiate tumorigenesis. A
critical question that remains unanswered is what factors and ecological
principles mediate the gut microbiome's influence on this process. Deciphering
how changes in community composition and structure disturb these interactions,
and subsequently modulate tumorigenesis, is an essential step in understanding
the etiology of CRC.

Several bacterial populations including *E. coli*, *Bacteroides fragilis*, and
*Fusobacteirum nucleatum* have been shown to directly influence the process of
tumorigenesis in the colon. The mechanisms by which bacteria potentiate these
processes range from the production of carcinogenic toxins [Arthur et al.,
2012; Sears et al., 2008] to direct manipulation of the inflammatory status in
a tumor's microenvironment [Kostic et al., 2013; Rubinstein et al., 2014].
Although individual bacterial populations undoubtedly modulate colorectal
cancer, there are likely a myriad of commensal bacteria that work together to
influence tumorigenesis in the colon. This is supported by several studies that
have explored the gut microbiome associated with individuals with CRC [Zackular
et al., 2014; Chen et al., 2013; Chen, et al., 2012; Kostic et al., 2012; Geng
et al., 2013; Shen et al., 2013; Sobhani et al., 2011; Wang et al., 2012; Ahn et
al., 2013; Baxter et al., 2014]. Each of these studies reported abnormal shifts
in the gut microbiome; however, there were no specific CRC-associated
bacterial populations that were consistently identified across all individuals
or studies. This is likely due to the fact that there is significant functional
redundancy within the gut microbiome and various bacterial populations may fill
similar roles in tumorigenesis [Lepage et al., 2011; Turnbaugh et al., 2009; Qin
et al., 2010]. Furthermore, some bacterial populations have been hypothesized be
protective against CRC [Louis and Flint, 2009; Arthur et al., 2011; Appleyard et
al., 2011]. This protective phenotype may be mediated through metabolite
production, induction of immunotolerance, or an ability to outcompete pathogenic
bacteria [Zhu et al., 2011]. We hypothesize that multiple bacteria in the gut
microbiome have the potential to play antagonistic or protective roles in
tumorigenesis; thus, the gut microbiome's influence on CRC is likely to be
polymicrobial.

Altering the composition of the microbiome in an inflammation-based mouse model
of CRC resulted in varying numbers of colonic tumors. We also treated
conventionally-raised mice with a cocktail of metronidazole, streptomycin, and
vancomycin in their drinking water while they received the tumor-inducing
treatment and observed a significant decrease in tumorigenesis [Zackular et al,
2013]. In the current study, we explored this result further by altering the
composition of this antibiotic cocktail to test the hypothesis that the gut
microbiome structure mediates tumor multiplicity and severity. To better
understand the composition of the microbiome and its dynamics, we extended this
result by permuting the composition of the antibiotic cocktail and associated
changes in the microbiome with the number of tumors that formed. These results
confirmed our hypothesis that the microbiome is capable of driving
tumorigenesis. To further test this hypothesis, we designed an antiboitic-based
intervention to determine whether it was possible to arrest tumorigenesis. Our
analysis supports a model by which individual bacterial populations play an
important role in CRC, but the ecological interactions and community structure
of the gut microbiome mediate the capacity to modulate tumorigenesis.


**Results**

```{r setup, cache=TRUE}
set.seed(6201976)
source("code/rf_baseline_analysis.R")

shared_file <- "data/process/ab_aomdss.trim.contigs.good.unique.good.filter.unique.precluster.pick.pick.pick.an.unique_list.0.03.subsample.shared"
rel_abund <- get_rel_abund(paste0(PROJHOME, "/", shared_file), "DSS")

treatment <- sapply(rownames(rel_abund), get_treatment)
eartag <- sapply(rownames(rel_abund), get_eartag)
day <- sapply(rownames(rel_abund), get_day)

baseline <- day == 0                        #baseline samples are from day 0
treatment_baseline <- treatment[baseline]   #update the metadata
eartag_baseline <- eartag[baseline]         #update the metadata
day_baseline <- day[baseline]               #update the metadata
rabund_baseline <- rel_abund[baseline,]     #extract the baseline samples

#what mice do we have baseline data for? necessary because the tumor data
#is indexed by `treatment_eartag`
mice_baseline <- paste(treatment_baseline, eartag_baseline, sep="_")

tumor_count.file <- paste0(PROJHOME, "/data/process/tumor_counts.tsv")
tumor_counts <- read.table(file=tumor_count.file, header=T)
tumor_counts_baseline <- tumor_counts[mice_baseline,]   #get the tumor count data
```

***Antibiotic perturbation of the gut microbiome modulates tumorigenicity.***
We subjected conventionally-raised C57BL/6 mice to an inflammation-based model
of colorectal cancer that utilizes azoxymethane (AOM) as a mutatgen and dextran
sodium sulfate (DSS) to induce inflammation [Zackular et al., 2013; Baxter et
al. 2014] (Figure 1A). As a test of the hypothesis that the microbiota are
involved in tumorigenesis, we manipulated the microbiota by administering eight
antibiotic treatment groups and then quantified the effects of the treatments on
the number of tumors observed at the end of the model. Specifically, we treated
mice with (i) no antibiotics, (ii) metronidazole, streptomycin, and vancomycin
(all antibiotics), (iii) streptomycin and vancomycin ($\Delta$ metronidazole),
(iv) metronidazole and vancomycin ($\Delta$ streptomycin), (v) metronidazole and
streptomycin ($\Delta$ vancomycin), (vi) metronidazole, (vii) streptomycin, and
(viii) vancomycin. Quantitative PCR targeting the 16S rRNA gene indicated that
the number of copies of the 16S rRNA gene did not differ between the treatment
groups. The three antibiotics were selected because they were thought to target
different groups of bacteria including anaerobes (metronidazole), Gram-negatives
(streptomycin), and Gram-positives (vancomycin). By sequencing the 16S rRNA
genes that were present in the feces of the mice that were treated with the
antibiotics it was clear that these treatments resulted in different starting
communities (Figure 1B); however, the composition of these communities was not a
predictable result of the antibiotic that was used to treat the mice. In support
of our hypothesis we observed that antibiotic treatment resulted in a decrease
in tumor multiplicity relative to the mice that did not receive antibiotics
(Figure 1CD). These results demonstrated that perturbation of the microbiota
through the use of antibiotics yields a differential capacity for colon
tumorigenesis.


```{r predict_tumor_baseline, cache=TRUE}
n_trees <- 10000 #number of trees for random forest
rf_baseline_thresh <- 0.015 #an average of 1.5%

rf_baseline_forest <- get_forest(rabund=rabund_baseline,
                            dependent=tumor_counts_baseline,
                            n_trees=n_trees,
                            threshold=rf_baseline_thresh)

rf_baseline_rabund <- filter_rabund(rabund_baseline, rf_baseline_thresh)

rf_simplify_rsq <- simplify_model(tumor_counts_baseline, rf_baseline_forest,
    rabund_baseline)
```


***The final tumor burden can be predicted from the initial microbiota.***
Because we obtained serial fecal samples it was possible to ascertain the
composition of the microbiota for each mouse as well as the number of tumors in
its colon at the end of the model. Using the 16S rRNA gene sequence data
generated from feces collected on the day of AOM injection, we assigned the
sequences to operational taxonomic units (OTUs) that were defined as a group of
sequences that, on average, were not more than 3% different from each other. We
then used the regression-based random forest machine learning algorithm to
identify OTUs that would enable us to predict the number of tumors that were
found in the same mice at the end of the model. The model that included OTUs
that had an average relative abundance greater than 1.5% resulted in the
greatest percentage of the variance explained (Supplementary Figure 1). This
model included `r get_n_otus(rf_baseline_forest)` OTUs and explained
`r round(100*get_rsq(rf_baseline_forest), 1)`% of the variation in the tumor counts (Figure 2).
As suggested in Figure 2A, when the OTUs were sorted in decreasing order by
their Gini coefficient, there was a jump between the sixth and seventh OTUs. In
fact, when we reconstructed the model using only the six OTUs that provided the
largest Gini coefficients, the model explained
`r round(100*rf_simplify_rsq[6], 1)`% of the variation in the tumor
counts. These six OTUs included members of the Firmicutes (OTU 6), Bacteroidetes
(OTUs 4 and 19), Proteobacteria (OTU 3), and Tenericutes (OTUs 34 and 35).
Increased numbers of tumors were associated with decreases in the relative
abundance of relatives of the Enterobacteriaceae (OTU 3), *Ureaplasma* (OTU 34),
and *Lactobacillus* (OTU 6) and increases in the relative abundance of the
*Anaeroplasma* (OTU 35), Porphyromonadaceae (OTU 4), and *Prevotella* (OTU 19).
Our random forest modeling demonstrated that it was possible to predict the
number of tumors at the end of the model based on the composition of the
microbiota at the beginning of the model.




```{r predict_tumor_final, cache=TRUE}
get_end_sample <- function(treat_et){
    treat_et_days <- day[grep(treat_et, names(day))]
    names(which.max(treat_et_days))
}

unique_treat_et <- unique(paste(treatment, eartag, sep="_"))
ends <- sapply(unique_treat_et, get_end_sample)

treatment_final <- treatment[ends]   #update the metadata
eartag_final <- eartag[ends]         #update the metadata
dayfinal <- day[ends]               #update the metadata
rabund_final <- rel_abund[ends, ]
tumor_counts_final <- tumor_counts[unique_treat_et, 1]

n_trees <- 10000 #number of trees for random forest
rf_final_thresh <- 0.015 #an average of 1.5%

rf_final_forest <- get_forest(rabund=rabund_final,
                            dependent=tumor_counts_final,
                            n_trees=n_trees,
                            threshold=rf_final_thresh)

rf_final_rabund <- filter_rabund(rabund_final, rf_final_thresh)

rf_simplify_rsq <- simplify_model(tumor_counts_final, rf_final_forest,
                                                                rabund_final)
```

***The final tumor burden can be predicted from the final microbiota.***
Similar to our analysis using the initial composition of the microbiota, we
developed a random forest regression model to predict the number of tumors in
the mice based on the composition of the microbiota at the end of the model. The
model included `r ncol(rf_final_rabund)` OTUs after we again applied a filter
requiring each OTU to have an average relative abundance of at least 1.5%. The
model explained `r round(100*get_rsq(rf_final_forest), 1)`% of the variation in
the tumor counts (Supplementary Figure 2), which is less than we observed when
we modeled tumor counts based on the initial community composition. The seven
most important OTUs in the model explained `r round(100*rf_simplify_rsq[7], 1)`%
of the variation and included *Odoribacter* (OTU 70), *Bacteroides* (OTU 5),
*Lactobacillus* (OTU 6), Enterobacteriaceae (OTU 3), *Alloprevotella* (OTU 14),
*Prevotella* (OTU 19), and Betaproteobacteria (OTU 17) (Supplementary Figure 3).
Interestingly, of the OTUs that were predictive of the number of tumor counts
using the baseline and final community composition data, only three of the OTUs
overlapped. These included *Lactobacillus* (OTU 6), Enterobacteriaceae (OTU 3),
and *Prevotella* (OTU 19). These results confirm those of our earlier study
where we used mice that did not receive antibiotics and the sequencing was
performed using the 454 platform [ref].


```{r distance_tumor_correlation}
get_dist_lt <- function(file_name){

    dist_vector <- scan(file_name, what="", sep="\n")

    n_samples <- as.numeric(dist_vector[1])
    dist_vector <- dist_vector[-1]

    sample_names <- character(n_samples)
    distance_matrix <- matrix(rep(0, n_samples*n_samples), nrow=n_samples)

    for(i in 1:n_samples){

        split_line <- unlist(strsplit(dist_vector[i], "\t"))
        sample_names[i] <- split_line[1]

        if(i > 1){
            distance_matrix[i,1:(i-1)] <- as.numeric(split_line[2:i])
        }
    }

    distance_matrix <- distance_matrix + t(distance_matrix)
    rownames(distance_matrix) <- sample_names
    colnames(distance_matrix) <- sample_names
    distance_matrix
}


get_start_end_dist <- function(u_treat_et, days){
    subset <- days[treat_et == u_treat_et]
    start <- names(subset[subset==0])
    end <- names(sort(subset, decreasing=TRUE))[1]
    distances[start, end]
}


dist_file <- "data/process/ab_aomdss.trim.contigs.good.unique.good.filter.unique.precluster.pick.pick.pick.an.unique_list.thetayc.0.03.lt.ave.dist"

distances <- get_dist_lt(dist_file)
treat_et <- gsub("_d.*", "", rownames(distances))
days <- sapply(rownames(distances), get_day)
start_end_dist <- unlist(sapply(unique(treat_et), get_start_end_dist, days))

tumor_counts_prepost <- tumor_counts[names(start_end_dist), "n_tumors"]
dist_tumor_cor <- cor.test(tumor_counts_prepost, start_end_dist,
                                method="spearman")
```

***The microbial community is dynamic.***
Using mice that were colonized with human feces, we observed that the the level
of tumor burden was associated with the amount of change in the community
structure over the course of the AOM-DSS model [ref]. In the current study,
however, there was a non-significant association between the change in the
community structure as measured by the \theta<sub>YC</sub> metric of community
structure similarity and tumor burden
(\rho=`r round(dist_tumor_cor$estimate, 2)`,
P=`r round(dist_tumor_cor$p.value, 2)`; Figure 4A). The mice that did not
receive antibiotics and those that received the $\Delta$ vancomycin and $\Delta$
metronidazole treatments changed the most over the course of the model. When we
investigated the temporal progression of the three OTUs that were important in
predicting the number of tumors based on the starting and final community
structure (i.e. OTUs 3, 6, and 19) we observed a dynamic progression across the
model. Not sure what else to say here...



***Antibiotic intervention narrows possible mechanisms of microbiome involvement
in tumorigenesis.***
The AOM-DSS model has been shown to reproduce the patterns seen in human CRC.
AOM induces DNA damage and the DSS treatments induces inflammation. To determine
whether the gut microbiome facilitates tumorigenesis by modulating AOM-induced
mutations or inflammation, we performed two antibiotic intervention experiments.
We first treated mice with the full antibiotic cocktail two weeks prior to the
administration of AOM and up until the first round of DSS (Figure 1A). We found
that these mice had a similar tumor burden to untreated mice (Figure 5). Next,
we treated mice before the second round of DSS administration, following the
initiation of AOM/DSS mediated tumorigenesis and inflammation (Figure 1A). In
this treatment, there was a significant decrease in the number of tumors (Figure
5). These results suggest that the gut microbiome mediated affect on CRC is
independent of AOM carcinogenesis. Furthermore, it shows that targeting the gut
microbiome at later stages of tumor growth is a viable option for minimizing
tumorigenesis and highlights microbiome manipulation as a potential therapeutic
in CRC.  


**Discussion**

In the present study, we established the importance of the microbial community
structure in mediating the gut microbiome’s capacity for tumorigenesis. We
demonstrated that manipulation of the murine gut microbiome with an assortment
of antibiotic treatments resulted in non-overlapping community structures with a
disparate level of tumorigenesis. Enrichment in the relative abundance of
several bacterial populations was associated with high and low levels of colonic
tumors. We determined that outgrowth of potentially inflammatory members of the
gut microbiome only mediated increased tumorigenesis when there was a
corresponding decrease in potentially protective, butyrate producing, bacteria.
By perturbing the community at various time points in the AOM/DSS model, we
determined that the gut microbiome is likely potentiating tumorigenesis
independent of AOM-carcinogenesis. Our experiments also demonstrated that
targeting the gut microbiome at the first signs of dysbiosis is a viable
strategy for the amelioration of colon tumorigenesis.

In recent years, there has been a focus on identifying bacterial populations
that are etiologic agents of CRC. Several commensal bacteria, including
*Fusobacterium nucleatum* and enterotoxigenic *Bacteroides fragilis* (ETBF)
have been linked to CRC in humans [Arthur et al., 2013; Rubinstein et al.; 2013,
Sears et al., 2008]. *F. nuceleatum* can manipulate the inflammatory environment
on in the tumor microenvironment in multiple intestinal neoplasia mice and in
the studied population has been detected on the surface of over 50% of adenomas
[Kostic et al., 2012; Kostic et al., 2013]. ETBF increases tumor multiplicity in
the colon of multiple intestinal neoplasia mice through the action of a secreted
metalloprotease toxin. It has been estimated that between 5-35% of people carry
ETBF [Housseau and Sears, 2010]. Although there is substantial evidence for a
role in potentiating tumorigenesis, the fact that each of these bacteria is only
associated with a fraction of CRCs suggests that it is unlikely that there is a
single microbial agent that causes cancer. Rather, the gut microbiome’s role in
CRC is likely polymicrobial. The results in the present study support this
hypothesis, as we demonstrated that non-overlapping community structures could
confer similar levels of tumorigenesis in mice. When we examined the relative
abundance of bacterial populations associated with increased tumor burden, we
never observed all three treatment groups with high tumor levels (vancomycin
only, streptomycin only, and ∆metronidazole) showing a consistent enrichment.
The same was observed with potentially protective populations across all
treatment groups that developed significantly less tumors (All antibiotics,
∆vancomycin, ∆streptomycin, and metronidazole only). This suggests that various
bacteria within the gut microbiome may confer the same function and be playing
redundant tumor-modulating roles.


During the time course of tumorigenesis we observed a marked increase in members
of the *Enterobacteriaceae* associated with two antibiotic treatment groups
(∆metranidazole and ∆vancomycin). Interestingly, one treatment group
(∆vancomycin) developed fewer tumors despite a similar increase in
this potentially tumor-modulating bacterial clade. A recent study by Arthur
and colleagues showed that in an IL-10-deficient colitis-associated mouse model
of CRC; there is an enrichment of *Enterobacteriaceae* associated with
inflammation [Arthur et al., 2012]. This leads to an expansion of *E. coli*
populations with genotoxic capabilities and a consequential increase in tumor
multiplicity and invasion. Furthermore, members of the Enterobacteriaceae have
been shown to perpetuate inflammation in several inflammatory diseases,
including ulcerative colitis, which increase an individual’s risk of developing
CRC [Rolhion and Darfeuille-Michaud, 2007; Garrett et al., 2007; Rooks et al.,
2014]. When we further examined the two antibiotic treatment groups, we observed
that mice with an increased tumor burden had a corresponding decrease in several
potentially anti-inflammatory and butyrate producing bacterial populations.
These observations support a model by which the pathogenicity of individual
members of the gut microbiome is mediated by the community structure and
ecological interactions within the gut microbiome. We hypothesize that
inflammatory and carcinogenic commensal bacteria, such as *Enterobacteriaceae,*
can only mediate a pathogenic phenotype if the context of the community
structure is conducive.

One mechanism, by which community structure likely mediates tumorigenicity, is
through shifts in the balance of immunomodulatory metabolites and signals.
During health, the gut microbiome is an important mediator of immunotolerance,
but when the balance of pro- and anti-inflammatory signals is disrupted gut
pathologies can arise [Kelly et al., 2005]. In our mice, Enterobacteriaceae is
likely acting as an inflammatory member of the gut microbiome. We only observed
an increase in tumorigenesis when there was a corresponding depletion of
potentially protective members of the genera *Clostridium, Enterococcus, and
Streptococcus*. Members of the *Clostridium* are known producers of short chain
fatty acids (SCFA) in the colon [Louis et al., 2009]. SCFA, specifically
butyrate, are important nutrients for colonocytes and they also possess
anti-inflammatory and anti-tumor properties [Louis et al., 2009]. Furthermore,
*Enterococcus* and *Streptococcus* species have been linked to down-regulating
the inflammatory response in the colon [Wang et al., 2008; Kaci et al, 2011]. It
is likely that these bacterial populations have the ability to antagonize
inflammatory clades (e.g. *Enterobacteriaceae*) and confer protection; however,
when perturbation to the microbial community structure disrupts this
homeostasis, these opportunistic pathogens can potentiate tumorigenesis

In our previous work, we demonstrated that dysbiosis of the gut microbiome
generates a pro-inflammatory environment which results in a self-reinforcing
pathogenic cascade between the gut microbiome and the host [Zackular et al.,
2013]. In this study we demonstrated that antibiotic manipulation of the gut
microbiome after the initiation of inflammation and tumorigenesis can
significantly decreased tumorigenesis in mice. This highlights the efficacy of
targeting the gut microbiome in CRC. Additional studies are needed to explore
the viability of manipulating the gut microbiome in CRC with methods such as
diet, probiotics, and prebiotics.




**Materials & Methods**

**Animals and animal care.** Studies were conducted using adult (8 to 12 week
old) age-matched C57BL/6 male mice that were maintained under SPF conditions.
Mice were co-housed in groups of five and fed the same autoclaved chow diet. All
animal experiments were approved by the University Committee on Use and Care of
Animals at the University of Michigan.

**Inflammation-induced colon tumorigenesis.** Mice received a single
intraperitoneal (i.p.) injection of azoxymethane (10 mg/kg). Water containing
2% DSS was administered to mice beginning on day 5 for 5 days followed by 16
days of water. This was repeated twice for a total of 3 rounds of DSS
[Zackular et al., 2013]. Mice were euthanized 3 weeks after the third round of
DSS administration for tumor counting. At necropsy, all colons were harvested,
flushed of luminal contents, and cut open longitudinally to count and measure
tumors.

**Antibiotic treatment.** Mice were treated with all possible combinations of
metronidazole (0.75 g/L), streptomycin (2 g/L), and vancomycin (0.5 g/L) to
create eight treatment groups: no antibiotics, all antibiotics (n=5)
(metronidazole, streptomycin, and vancomycin), $\Delta$ metronidazole (n=5)
(streptomycin and vancomycin), $\Delta$ streptomycin (n=5) (metronidazole and
vancomycin), $\Delta$ vancomycin (n=5) (metronidazole and streptomycin), metronidazole
only, streptomycin only, and vancomycin only (n=3). Antibiotics were
administered in mouse drinking water for 2 weeks prior to and throughout the
duration of AOM/DSS administration, unless otherwise specified in Figure 1A.
Tumors were enumerated at the end of the model.

**16S rRNA quantitative PCR (qPCR) analysis.** Relative bacterial loads were
quantified by qPCR analysis of bacterial genomic DNA using KAPA SYBR-fast
Master Mix (KAPA biosciences) and universal 16S rRNA gene primers (F:
ACTCCTACGGGAGGCAGCAGT; R: ATTACCGCGGCTGCTGGC) [Vaishnava et al., 2011].
Samples were normalized to fecal mass and relative fold change was determined
using untreated stool samples for each replicate mouse. Note that qPCR
measures relative fold change of 16S gene copy number, not actual bacterial
numbers.

**DNA extraction and 16S rRNA gene sequencing** Fecal samples were collected
daily from the mice throughout the AOM/DSS protocol and immediately frozen for
storage at -20°C. For each mouse, 8 fecal samples distributed over the 73-day
timeline of the AOM/DSS model were selected for analysis (Figure 1A). Microbial
genomic DNA was extracted using the PowerSoil-htp 96 Well Soil DNA Isolation Kit
(MO BIO laboratories) using an EpMotion 5075. The V4 region of the 16S rRNA gene
from each sample was amplified and sequenced using the Illumina MiSeq Personal
Sequencing platform as described elsewhere [Kozich et al., 2013]. Sequences were
curated as described previously using the mothur software package [Schloss et
al., 2009]. Briefly, we reduced sequencing and PCR errors, aligned the resulting
sequences to the SILVA 16S rRNA sequence database [Pruesse et al., 2007], and
removed any chimeric sequences flagged by UCHIME [Edgar, 2011]. Sequences had a
median length of 253 bp and we rarefied to 2,500 sequences per sample to limit
effects of uneven sampling. The complete analysis methods and this document as
an R-executable document are available at
https://github.com/SchlossLab/Zackular_AbAOMDSS_GutMicrobes_2015. All FASTQ
sequence data can be obtained from the Sequece Read Archive at NCBI (Need
to deposit data).


**Acknowledgements**

This work was supported by grants from the National Institutes for Health to PDS
(R01GM095356, R01HG005975, P30DK034933, University of Michigan GI SPORE) and GYC
(University of Michigan GI SPORE and ARRA Supplement P30CA4659-22S3).



**References**

1. Backhed F, Ley RE, Sonnenburg JL, Peterson DA, Gordon JI (2005)
    Host-bacterial mutualism in the human intestine. Science 307:
    1915-1920.

2. Chen HM, Yu YN, Wang JL, Lin YW, Kong X, et al. (2013) Decreased
    dietary fiber intake and structural alteration of gut microbiota in
    patients with advanced colorectal adenoma. Am J Clin Nutr.

3. Chen W, Liu F, Ling Z, Tong X, Xiang C (2012) Human Intestinal Lumen
    and Mucosa-Associated Microbiota in Patients with Colorectal Cancer.
    PLoS ONE 7(6): e39743. doi:10.1371/journal.pone.0039743

4. Kostic AD, Gevers D, Pedamallu CS, Michaud M, Duke F, et al. (2012)
    Genomic analysis identifies association of *Fusobacterium* with
    colorectal carcinoma. Genome Res 22: 292-298.

5. Geng J, Fan H, Tang X, Zhai H, Zhang Z (2013) Diversified pattern of
    the human colorectal cancer microbiome. Gut Pathog 5: 2.

6. Shen XJ, Rawls JF, Randall T, Burcal L, Mpande CN, et al. (2010)
    Molecular characterization of mucosal adherent bacteria and
    associations with colorectal adenomas. Gut Microbes 1: 138-147.

7. Sobhani I, Tap J, Roudot-Thoraval F, Roperch JP, Letulle S, et al.
    (2011) Microbial dysbiosis in colorectal cancer (CRC) patients. Plos
    One 6: e16393.

8. Wang T, Cai G, Qiu Y, Fei N, Zhang M, et al. (2012) Structural
    segregation of gut microbiota between colorectal cancer patients and
    healthy volunteers. ISME Journal 6: 320-329.

9. Ahn J, Sinha R, Pei ZH, Dominianni C, Wu J, et al. (2013) Human Gut
    Microbiome and Risk for Colorectal Cancer. Jnci-J Natl Cancer Inst
    105: 1907-1911.

10. Zackular JP, Baxter NT, Iverson KD, Sadler WD, Petrosino JF, et al.
    (2013) The Gut Microbiome Modulates Colon Tumorigenesis. Mbio 4.

11. Arthur JC, Perez-Chanona E, Muhlbauer M, Tomkovich S, Uronis JM, et
    al. (2012) Intestinal inflammation targets cancer-inducing activity
    of the microbiota. Science 338: 120-123.

12. Yue JC, Clayton MK (2005) A similarity measure based on species
    proportions. Communications in Statistics-Theory and Methods 34:
    2123-2131.

13. Sears CL, Islam S, Saha A, Arjumand M, Alam NH, et al. (2008)
    Association of enterotoxigenic Bacteroides fragilis infection with
    inflammatory diarrhea. Clin Infect Dis 47: 797-803.

14. Kostic AD, Chun E, Robertson L, Glickman JN, Gallini CA, et al.
    (2013) Fusobacterium nucleatum potentiates intestinal tumorigenesis
    and modulates the tumor-immune microenvironment. Cell Host Microbe
    14: 207-215.

15. Rubinstein MR, Wang X, Liu W, Hao Y, Cai G, et al. (2013)
    Fusobacterium nucleatum Promotes Colorectal Carcinogenesis by
    Modulating E-Cadherin/≤-Catenin Signaling via its FadA Adhesin. Cell
    Host Microbe 14: 195-206.

16. Louis P, Flint HJ (2009) Diversity, metabolism and microbial ecology
    of butyrate-producing bacteria from the human large intestine. FEMS
    Microbiology Letters 294: 1-8.

17. Arthur JC, Uronis JM, Chanona EP, Keku TO, Muehlbauer M, et al.
    (2011) Inflammation Impacts the Development of Colorectal Cancer
    Through Changes in Microbial Composition and Associated Genotoxic
    Capacity. Gastroenterology 140: S181-S181.

18. Appleyard CB, Cruz ML, Isidro AA, Arthur JC, Jobin C, et al. (2011)
    Pretreatment with the probiotic VSL\#3 delays transition from
    inflammation to dysplasia in a rat model of colitis-associated
    cancer. Am J Physiol Gastrointest Liver Physiol 301: G1004-G1013.

19. Zhu Y, Michelle Luo T, Jobin C, Young HA (2011) Gut microbiota and
    probiotics in colon tumorigenesis. Cancer Lett 309: 119-127.

20. Zackular, JP, Rogers, MA, Rufin, MT, and Schloss, PD. (2014) The
    Human Gut Microbiome as a Screening Tool for Colorectal Cancer.
    Cancer Prev Res*,* doi: 10.1158/1940-6207.CAPR-14-0129

21. Lepage P, Leclerc MC, Joossens M, Mondot S, Blottire HM, et al.
    (2012) A metagenomic insight into our gut's microbiome. Gut.

22. Turnbaugh PJ, Ridaura VK, Faith JJ, Rey FE, Knight R, et al. (2009)
    The Effect of Diet on the Human Gut Microbiome: A Metagenomic
    Analysis in Humanized Gnotobiotic Mice. Sci Transl Med 1.

23. Qin J, Li R, Raes J, Arumugam M, Burgdorf KS, et al. (2010) A human
    gut microbial gene catalogue established by metagenomic sequencing.
    Nature 464: 59-65.

24. Levy R, Borenstein E (2013) Metabolic modeling of species
    interaction in the human microbiome elucidates community-level
    assembly rules. Proc Natl Acad Sci U S A 110: 12804-12809.

25. Marino S, Baxter NT, Huffnagle GB, Petrosino JF, Schloss PD (2014)
    Mathematical modeling of primary succession of murine intestinal
    microbiota. Proc Natl Acad Sci U S A 111: 439-444.

26. Lepp PW, Brinig MM, Ouverney CC, Palm K, Armitage GC, et al. (2004)
    Methanogenic Archaea and human periodontal disease. Proc Natl Acad
    Sci U S A 101: 6176-6181.

27. Turnbaugh PJ, Ley RE, Mahowald MA, Magrini V, Mardis ER, et al.
    (2006) An obesity-associated gut microbiome with increased capacity
    for energy harvest. Nature 444: 1027-1031.

28. Tamboli CP, Neut C, Desreumaux P, Colombel JF (2004) Dysbiosis in
    inflammatory bowel disease. Gut 53: 1-4.

29. Saulnier DM, Riehle K, Mistretta TA, Diaz MA, Mandal D, et al.
    (2011) Gastrointestinal Microbiome Signatures of Pediatric Patients
    With Irritable Bowel Syndrome. Gastroenterol 141: 1782-1791.

30. Arthur JC, Jobin C (2013) The complex interplay between
    inflammation, the microbiota and colorectal cancer. Gut Microbes 4.

31. Housseau F, Sears CL (2010) Enterotoxigenic Bacteroides fragilis
    (ETBF)-mediated colitis in Min (Apc(+/-)) mice A human
    commensal-based murine model of colon carcinogenesis. Cell Cycle 9:
    3-5.

32. Rolhion N, Darfeuille-Michaud A (2007) Adherent-invasive Escherichia
    coli in inflammatory bowel disease. Inflammatory Bowel Diseases 13:
    1277-1283.

33. Garrett WS, Lord GM, Punit S, Lugo-Villarino G, Mazmanian SK, et al.
    (2007) Communicable ulcerative colitis induced by T-bet deficiency
    in the innate immune system. Cell 131: 33-45.

34. Rooks MG, Veiga P, Wardwell-Scott LH, Tickle T, Segata N, et al.
    (2014) Gut microbiome composition and function in experimental
    colitis during active disease and treatment-induced remission. Isme
    Journal.

35. Louis P, Flint HJ (2009) Diversity, metabolism and microbial ecology
    of butyrate-producing bacteria from the human large intestine. Fems
    Microbiology Letters 294: 1-8.

36. Wang SG, Ng LHM, Chow WL, Lee YK (2008) Infant intestinal
    Enterococcus faecalis down-regulates inflammatory responses in human
    intestinal cell lines. World Journal of Gastroenterology 14:
    1067-1076.

37. Kaci G, Lakhdari O, Dore J, Ehrlich SD, Renault P, et al. (2011)
    Inhibition of the NF-kappa B Pathway in Human Intestinal Epithelial
    Cells by Commensal Streptococcus salivarius. Applied and
    Environmental Microbiology 77: 4681-46


\newpage


**Figure legends**

**Figure 1. Antibiotic perturbation drives changes in microbial community
structure and final tumor burden.** The AOM-DSS model was applied to C57BL/6
mice reared under standard pathogen free (SPF) conditions with different
antibiotic perturbations; Black arrows indicate fecal samples that used for our
analysis (A). The mice were treated with all possible combinations of
metronidazole, streptomycin, and vancomycin to create eight treatment groups,
which resulted in considerable differences in the taxonomic composition of the
gut communities at the start of the model (Day 0) (B). These communities
resulted in a continuum of tumor burden in the mice (C and D).  

```{r figure1, fig.width=7.5, fig.height=10}
source("code/build_figure1.R")
```


**Figure 2. A random forest model successfully predicted the number of tumors in
the mice at the end of the model (A) based on their microbiota composition at the
start of the model (B).** The OTUs in B are ranked in decreasing order of their
Gini coefficient. The relationships between the first 6 OTUs and the number of
tumors found in those mice are shown in Figure 3.

```{r figure2, fig.width=7.5, fig.height=3.5}
layout(matrix(c(1,2), nrow=1), width=c(1,1.2), height=0.8)
plot_forest_fit(tumor_counts_baseline, rf_baseline_forest, rf_baseline_rabund, treatment_baseline)
mtext(text="A", line=1.5, side=2, at=23, las=2, cex=2, font=2)
plot_importance(rf_baseline_forest)
mtext(text="B", line=8.5, side=2, at=15, las=2, cex=2, font=2)
```


**Figure 3. Relationship between the initial relative abundance of the most
informative OTUs from the random forest model with the number of tumors found
in the mice at the end of the model.** The vertical gray line indicates the
limit of detection.

```{r figure3, fig.width=7.5, fig.height=5}
plot_baseline_features(tumor_counts_baseline, rf_baseline_forest,
                                        rf_baseline_rabund, treatment_baseline)
```



**Figure 4. The murine microbiota is dynamic but the amount of change is
not associated with the final number of tumors.** The structure of the gut
microbiota associated with untreated and the $\Delta$ metronidazole and $\Delta$
vancoymcin-treated mice changed the most throughout the model as measured using
the $\Theta$<sub>YC</sub> distance metric (A). OTUs 3, 6, and 19 were among the
most salient features for predicting tumor burden at the beginning and end of
the model (B). The plotting symbols and characters are the same as those used in
Figure 1. In panel B, the median relative abundance is indicated by the plotting
symbol and the range of observed relative abundances is plotted by the vertical
bar. The vertical blue regions indicate when the DSS treatments were applied.

```{r figure4, fig.width=3.75, fig.height=7}
design <- matrix(c(2,1,
                    6,3,
                    6,4,
                    6,5,
                    0,7), byrow=T, nrow=5)
z <- layout(design, width=c(0.15,1), height=c(1.5,1,1,1,0.3))

par(mar=c(3,0.5,0.5,0.5))
treatments <- gsub("_.*", "", names(start_end_dist))
plot(tumor_counts_prepost~start_end_dist, ylim=c(0,25), xlim=c(0,1),
    col=clrs[treatments],
    pch=pch[treatments], axes=FALSE, xlab="", ylab="")
mtext(side=1, text="Distance between communities on days 0 and 74", line=2, cex=0.7)
axis(1)
axis(2, las=2)
box()

#labels_no_n <- gsub(" \\(.*\\)", "", labels)
#legend(x=-0.025, y=26, legend=labels_no_n, pch=pch[names(labels)],
#        col=clrs[names(labels)], cex=0.9, bty="n")

par(mar=c(0,0,0,0))
plot(NA, xlim=c(0,1), ylim=c(0,1), axes=F, xlab="", ylab="", xpd=TRUE)
text(x=0.3, y=0.58, label="Number of tumors", srt=90, cex=1.5)
text(x=0.3, y=0.98, label="A", srt=0, cex=2, font=2)


par(mar=c(0.5,0.5,0.5,0.5))
source("code/make_timeline_plot.R")

par(mar=c(0,0,0,0))
plot(NA, xlim=c(0,1), ylim=c(0,1), axes=F, xlab="", ylab="", xpd=TRUE)
text(x=0.3, y=0.5, label="Relative abundance (%)", srt=90, cex=1.5)
text(x=0.3, y=1, label="B", srt=0, cex=2, font=2)

plot(NA, xlim=c(0.1,1), ylim=c(0,1), axes=F, xlab="", ylab="")
text(x=0.55, y=0.25, label="Days since AOM administration")
```


**Figure 5. Antibiotic intervention prior to second administration of
DSS alleviates tumor burden.** Interventions with an antibiotic cocktail
of metronidazole, vancomycin, and streptomycin were performed as
depicted in Figure 1A with enumeration of tumors performed at the end point of
the model (A). Representative images of tumors in the distal colon of mice from
each treatment group (B).

```{r figure5, fig.width=3.5, fig.height=3.5}
abx <- read.table(file="data/process/tumor_counts.tsv", header=T)
intervention <- read.table(file="data/process/intervention_tumor_counts.tsv", header=T)

layout(matrix(c(1,2), byrow=T, nrow=1), width=c(1.5,1.20), height=1)

par(mar=c(6,4,0.5,0.25))

plot(NA, xlim=c(0.5,4.5), ylim=c(0,25), xlab="", ylab="Number of tumors", axes=F)
stripchart(at=1, x=abx[grepl("NoAbs", rownames(abx)), "n_tumors"],
    vertical=T, method="jitter", col="black", pch=1, jitter=0.15, cex=1.0, add=T)
stripchart(at=2, x=abx[grepl("AllAbs", rownames(abx)), "n_tumors"],
    vertical=T, method="jitter", col="black", pch=19, jitter=0.15, cex=1.0, add=T)
stripchart(at=3, x=intervention[intervention$experiment=="Intervention1", "n_tumors"],
    vertical=T, method="jitter", col="black", pch=18, jitter=0.15, cex=1.5, add=T)
stripchart(at=4, x=intervention[intervention$experiment=="Intervention2", "n_tumors"],
    vertical=T, method="jitter", col="black", pch=17, jitter=0.15, cex=1.5, add=T)

text(y=-9, x=(1:4)-0.6, label=c("No antibiotics", "All antibiotics", "Intervention 1", "Intervention 2"), srt=70, cex=0.8, xpd=TRUE)
axis(2, las=1)
axis(1, tick=TRUE, labels=FALSE)
box()
mtext(text="A", side=2, line=2.5, at=25, font=2.5, cex=2, las=2, xpd=TRUE)


par(mar=c(6,1.5,0.0,0.0))
plot(c(0, 150), c(0, 150), type = "n", xlab = "", ylab = "", axes=FALSE)
img <- readPNG("results/pictures/tumor_images_intervention.png")
lim <- par()
rasterImage(img, -10, -10, 160, 160, xpd=TRUE)
text(label="B", x=-30.0, y=143, font=2.5, cex=2, xpd=TRUE)
```


**Supplementary Figure 1. Quality of random forest regression fit as a function
of the minimum average relative abundance an OTU must have to be included in the
model.** The integers displayed across the plot indicate the number of OTUs that
were included in the model. Because a minimum average relative abundance of 1.5%
yielded the best R<sup>2</sup>, it was used for the remainder of the analysis.

```{r supp_figure1, fig.width=5.0, fig.height=5.0}
threshold <- seq(0,0.03, 0.005)
n_trees <- 10000 #number of trees for random forest

rf_thresholds <- lapply(X=threshold, FUN=get_forest, rabund=rabund_baseline,
                         dependent=tumor_counts_baseline, n_trees=n_trees)

thresh_rsq <- sapply(rf_thresholds, get_rsq)

max_index <- which.max(thresh_rsq)
rf_baseline_forest <- rf_thresholds[[max_index]]    #get the forest
rf_baseline_thresh <- threshold[max_index]          #get the threshold that was used

#get filtered rabund table
rf_baseline_rabund <- filter_rabund(rabund_baseline, rf_baseline_thresh)

par(mar=c(5,5,0.5,0.5))
plot(thresh_rsq~threshold, pch=19, xlab="Minimum average relative abundance (%)",
                axes=FALSE, ylab=expression(R^2),
                ylim=c(0.4, 1.05*max(thresh_rsq)))
text(x=threshold, y=0.45, label=sapply(rf_thresholds, get_n_otus), cex=0.75)
axis(1, at=threshold, label=format(100*threshold, 1))
axis(2, las=2)
box()
```


**Supplemental Figure 2. A random forest model successfully predicted the number of tumors in
the mice at the end of the model (A) based on their microbiota composition at the
start end of the model (B).** The OTUs in B are ranked in decreasing order of their
Gini coefficient. The relationships between the first 6 OTUs and the number of
tumors found in those mice are shown in Supplemental Figure 3.

```{r supp_figure2, fig.width=7.5, fig.height=3.5}
layout(matrix(c(1,2), nrow=1), width=c(1,1.2), height=0.8)
plot_forest_fit(tumor_counts_final, rf_final_forest, rf_final_rabund, treatment_final)
mtext(text="A", line=1.5, side=2, at=23, las=2, cex=2, font=2)
plot_importance(rf_final_forest)
mtext(text="B", line=8.5, side=2, at=11, las=2, cex=2, font=2)
```


**Supplemental Figure 3. Relationship between the initial relative abundance of the most
informative OTUs from the random forest model with the number of tumors found
in the mice at the end of the model.** The vertical gray line indicates the
limit of detection.

```{r supp_figure3, fig.width=7.5, fig.height=5}
plot_final_features(tumor_counts_final, rf_final_forest, rf_final_rabund, treatment_final)
```
